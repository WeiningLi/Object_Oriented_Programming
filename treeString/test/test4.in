ra "abc"
ra "def"
ra "g"
cba
pb
pa
rb "abd"
rb "akjbfwkjeneflwkenfk"
+bbb
pb
db
rb "a b ?"
+bbb
+bbb
+bbb
+bab
db
pb
q
