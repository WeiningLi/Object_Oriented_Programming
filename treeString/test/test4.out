g
g
akjbfwkjeneflwkenfkakjbfwkjeneflwkenfk
"akjbfwkjeneflwkenfk" size: 19
 "akjbfwkjeneflwkenfk" size: 19
"g" size: 1
 "a b ?" size: 20
  "a b ?" size: 10
   "a b ?" size: 5
   "a b ?" size: 5
  "a b ?" size: 10
   "a b ?" size: 5
   "a b ?" size: 5
    "a b ?" size: 5
ga b ?a b ?a b ?a b ?a b ?a b ?a b ?a b ?
